test/e2e/aks.go

```diff
@@ -1,3 +1,4 @@
+//go:build e2e
```

Comment:
Is this build tag supposed to be here? I used to have an issue w/ Go 1.17 where it would add it when it runs the save actions.

Reply:
now that we build with go 1.17 it should be ok to have these there.  They are present on other files in this path already: https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/6a9300696954b36e032c26356d85bbb7487f32b5/test/e2e/e2e_suite_test.go#L1-L2

Reply:
I would go further and say that we *must* include these build tags